docs-ref-mapping/reference-unified.yml

```diff
@@ -321,6 +321,10 @@
       children:
       - '@azure-rest/ai-translation-text'
     - name: Anomaly Detector
+      href: ~/docs-ref-services/{moniker}/ai-anomaly-detector-rest-readme.md
+      children:
+      - '@azure-rest/ai-anomaly-detector'
+    - name: Anomaly Detector (deprecated)
```

Comment:
We alreay have "Anomaly Detector (deprecated)" at line 547 https://github.com/MicrosoftDocs/azure-docs-sdk-node/pull/1514/files#diff-594596e7bb045eeccea024715d4bb660b147569f1de067a3ed9d27d7a8dc4992R347